misc/python/materialize/output_consistency/data_generation/picker.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import random
+
+from materialize.output_consistency.expressions.expression import ExpressionWithNArgs
+
+
+def random_seed() -> int:
```

Comment:
General idea: We could run this test with one or multiple _fixed seeds_ to achieve deterministic executions in non-nightly executions. We could run this test _additionally with random seeds_ in nightly builds.